randovania/games/dread/json_data/Cataris.txt

```diff
@@ -3870,6 +3878,13 @@ Extra - asset_id: collision_camera_053
                   Heat/Cold Runs (Advanced) and Heat Damage ≥ 200
   > Dock to Underlava Puzzle Room 1
       Morph Ball and After Cataris - Underlava Speed Blocks Destroyed
+  > Event - Speed Blocks Destroyed
+      All of the following:
+          Gravity Suit and Speed Booster
+          Any of the following:
+              # Can short boost (or use a Speed Upgrade) from the top floor and carry Shinespark to Morph Tunnel below
+              Speed Booster Upgrade
+              Short Boost (Intermediate) and Can Short Boost
```

Comment:
This would require to remove the grapple block first. It is missing an after event grapple block?

Reply:
I seem to recall there being some convoluted logic path where it still guaranteed you needed grapple, but it probably can't hurt to put an After condition here.

Reply:
Underlava Puzzle 2 only needs grapple for that block. you could, in theory, go all the way around to kraid's area instead of just using the grapple door that is right there. This is in early Cataris, so there aren't any grapple requirements around really.